platform/core/src/services/UIModalService/index.js

```diff
@@ -38,19 +35,16 @@ function createUIModalService() {
  * Show a new UI modal;
  *
  * @param {Modal} component React component
- * @param {ModalProps} props { header, footer, backdrop, keyboard, show, closeButton, title, customClassName }
+ * @param {ModalProps} props { shouldCloseOnEsc, isOpen, closeButton, title, customClassName }
  */
 function show(
   component,
   props = {
-    header: null,
-    footer: null,
-    backdrop: false,
-    keyboard: false,
-    show: true,
+    shouldCloseOnEsc: false,
+    isOpen: true,
     closeButton: true,
     title: null,
-    customClassName: null,
+    customClassName: '',
```

Comment:
Why empty string? When you use `classnames()`, you can toss `null` in and everything works alright. Easier to check than empty string as well

Reply:
Done, thx.